src/Components/Notes/NoteCard.jsx

```diff
@@ -127,6 +137,27 @@ export default function NoteCard({
     const pauseTimeMs = 5000
     setTimeout(() => setSnackMessage(null), pauseTimeMs)
   }
+
+
+  /**
+   * deletes the note
+   *
+   * @param {string} repository
+   * @param {string} accessToken
+   * @param {number} noteNumber obtained from github issue
+   */
+  function deleteNote(repository, accessToken, noteNumber) {
+    if (deletedNotes !== null) {
```

Comment:
Can we remove the local tracking of deleted notes?

Reply:
I tested, the way it feels when this is removed, in a separate branch. 
The results are not as smooth, especially when the connection is slower.

I propose to leave it in,  
if we decide to change it in the future we can always remove it.